lib/helper/FileSystem.js

```diff
@@ -98,6 +106,14 @@ class FileSystem extends Helper {
     const content = getFileContents(this.file, encoding);
     fileEquals(this.file).negate(text, content);
   }
+
+  /**
+  * Returns filenames without directories.
+  */
+  getFileNames() {
```

Comment:
this method will be added to I object. 
Can we move it from class? 
Or will need rename method on grab*

Reply:
I'd like it to be added to I object. Getting filenames gives freedom for you to implement more scenario varieties like validating downloaded file count, formats etc. 
Also at first I thought that grab* prefix must be about getting something from web gui. I just wanted to separate those two concepts by not naming this as grabbing. So if you think low level things like getting filenames from local disk also should be named as grabbing, I can change it, What do you think?

Reply:
@DavertMik We need your help =)

Reply:
@DavertMik Any word about this mister?

Reply:
https://codecept.io/helpers/FileSystem#docsNav

I think `grab*` prefix would make it consistent across the helper methods.

Reply:
Done.